data/en/banner.yml

```diff
@@ -3,7 +3,11 @@ banner:
   enable : true
   title : "Karmada"
   content : "Open, Multi-Cloud, Multi-Cluster Kubernetes Orchestration"
-  button:
+  button_gh:
     enable : true
-    label : "GitHub"
+    label : "Learn More"
     link : "https://github.com/karmada-io/karmada/"
+  button_docs:
+    enable : true
+    label : "Getting Started"
+    link : "https://github.com/karmada-io/karmada/tree/master/docs"
```

Comment:
Why not let the `Learn More` link to `https://github.com/karmada-io/karmada/tree/master/docs`? 
And don't need to add another button here.

Reply:
Got that. but as we have `Learn More` takes you to `karmada-io` whole repo to choose any one further and `Getting started` gives you idea to find the docs related things, it differentiates both the links. 😕

Reply:
Does that make sense? If not I can go ahead change as per your suggestion. 💯

Reply:
Just let the icon(GitHub icon) link to `karmada-io/karmada` repo, and the `Learn More` button link to docs would be great!